servers/src/main/java/tachyon/master/file/journal/InodeFileEntry.java

```diff
@@ -103,7 +135,12 @@ public boolean isCacheable() {
   }
 
   @JsonGetter
-  public long getTtl() {
+  public long getTTL() {
     return mTTL;
   }
+
+  @Override
+  public JournalEntryType getType() {
```

Comment:
I was not really sorting the getters, but for consistency across different journal types, I moved the getType() to the bottom. I can go ahead and sort all getters alphabetically if you'd prefer that.